Formula/keptn.rb

```diff
@@ -0,0 +1,22 @@
+class Keptn < Formula
+  desc "Is the CLI for keptn.sh a message-driven control-plane for application delivery"
+  homepage "https://keptn.sh"
+  url "https://github.com/keptn/keptn/archive/0.7.3.tar.gz"
+  sha256 "7c5ab4bce7f8c75371a6130ae7929a9c7a88d05f29b96604b3849a93d2177228"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    Dir.chdir("cli")
+    system "go", "build", "-ldflags", "-s -w -X main.Version=#{version} -X main.KubeServerVersionConstraints=\"\"", "-trimpath", "-o", bin/"keptn"
```

Comment:
Any clue where `#{version}` is coming from? Or should this be fixed to 0.7.3?

Reply:
The version is autodetected from the URL and/or tag so only needs to be declared if it cannot be autodetected correctly.  

For more info look at:

https://rubydoc.brew.sh/Formula.html#stable-class_method

Reply:
Thanks for explaination :)